lobby-db/.include/prod_deploy.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+
+TASK=$1
+
+set -eu
+
+case $TASK in
+  flywayMigrate)
+     ;;
+  flywayInfo)
+     ;;
+   *)
+     echo "Param must be a one of 'flywayInfo' or 'flywayMigrate', it was: '$TASK'"
+     exit 1
+esac
+
+
+CREDS_FILE=~/lobby.db.creds
```

Comment:
Probably want to create this file with a leading period so it doesn't pollute a user's home directory listing.  Also, you may want to consider adding a _triplea-_ prefix (or placing it in the _~/triplea_ folder) so it's obvious it is related to TripleA in a sea of otherwise unrelated files.

Reply:
Shared account of TripleA user should have almost no files, pollution should not be a problem. It's a good consideration, for a developer rig I'd agree, for prod server I think the explicit visibility is also a benefit.

The folder namespacing is not a bad suggestion, but it either requires us to create the folder or to install lobby first. For now I'd like to start simple. After first prod deployment I expect us to update some of these scripts to match what really works out in the end.